libs/auth/scopes/src/lib/endorsements.scope.ts

```diff
@@ -0,0 +1,3 @@
+export enum EndorsementsScope {
+  actor = '@island.is/endorsements',
```

Comment:
Let's change this to `main`. We did the same in another PR, and `actor` may cause misunderstandings since the delegation system uses the word `actor` a lot.